packages/rich-text/src/plugins/EmbeddedEntityBlock/index.tsx

```diff
@@ -66,12 +67,14 @@ export const createEmbeddedAssetBlockPlugin = createEmbeddedEntityPlugin(BLOCKS.
 export const withEmbeddedEntryBlockOptions: CustomSlatePluginOptions = {
   [BLOCKS.EMBEDDED_ENTRY]: {
     type: BLOCKS.EMBEDDED_ENTRY,
+    // @ts-expect-error
```

Comment:
what's the issue with these references now?

Reply:
The typing of components expects the props to be either `RenderElementProps` or `RenderLeafProps` however, for some components we expect only elements. I wasn't sure what to do in these cases do so decided to push the changes and see what others think.